backend/resume_evaluations/service.py

```diff
@@ -0,0 +1,18 @@
+from backend.core.gemini import generate_structured_output
+
+from .exceptions import ResumeEvaluationError
+from .prompts import RESUME_EVAL_PROMPT
+from .schemas import ResumeEvaluationResponse
```

Comment:
The codebase convention is to use absolute import paths, rather than relative import paths.

```suggestion
from backend.resume_evaluations.exceptions import ResumeEvaluationError
from backend.resume_evaluations.prompts import RESUME_EVAL_PROMPT
from backend.resume_evaluations.schemas import ResumeEvaluationRequest, ResumeEvaluationResponse
```